examples/slider/slider-temperature.html

```diff
@@ -0,0 +1,308 @@
+      <div id="ex1" class="example">
+        <div class="slider-temperature">
+          <div id="id-temp-label" class="label">Temperature</div>
+
+          <svg role="none" class="slider-group"  width="145" height="360">
+            <text class="temp-value" x="28" y="35" >25°C</text>
+            <rect class="rail" x="60" y="47" width="8" height="300" rx="5"  aria-hidden="true"/>
+            <g role="slider"
+              id="id-temp-slider"
+              aria-orientation="vertical"
+              tabindex="0"
+              aria-valuemin="10.0"
+              aria-valuenow="25.0"
+              aria-valuetext="25.0 degrees Celsius"
+              aria-valuemax="38.0"
+              aria-labelledby="id-temp-label">
+              <text class="value" x="94" y="150">25°C</text>
+              <rect class="focus-ring"
+                x="35" y="170" width="105" height="24" rx="12" />
+              <rect class="thumb"
+                 x="35" y="145" width="48" height="14" rx="5"/>
+            </g>
+          </svg>
+        </div>
+      </div>
+      <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+    <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>The display of the slider's current value remains adjacent to the thumb as the thumb is moved, so people with a small field of view (e.g., due to magnification) can easily see the value while focusing on the thumb as they move it.</li>
+        <li>
+          To ensure the borders of the slider rail, thumb and focus ring have sufficient contrast with the background when high contrast settings invert colors, the color of the borders are synchronized with the color of the text content.
+          For example, support for using the high contrast foreground color for the focus ring border is achieved by using the CSS <code>currentColor</code> value for the <code>stroke</code> property of an inline SVG <code>rect</code> element.  To enable the high contrast background color to be the used as the contrasting color the <code>fill-opacity</code> attribute of the <code>rect</code> is set to zero.
+          If specific colors were instead used to specify the <code>stroke</code> and <code>fill</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the rail and the thumb or even make them invisible if their color matched the high contrast mode background.<br>
+          Note: The SVG element needs to have the CSS <code>forced-color-adjust</code> property set to <code>auto</code> for the <code>currentColor</code> value to be updated in high contrast mode.
+          Some browsers do not use <code>auto</code> for the default value.
+        </li>
+      </ul>
+    </section>
+
+    <section>
+      <h2 id="kbd_label" style="clear: both;">Keyboard Support</h2>
```

Comment:
@jongund 

What does the styling on this heading do? We don't have it on the kb heading in other examples? Should it be removed?

Reply:
@mcking65 
I think some older examples may use the CSS `float` property, so this is probably a ruminant from some of those early examples to make sure the heading was on a new line.   It can be removed.